mod_reforged/hooks/config/character_names.nut

```diff
@@ -242,3 +242,71 @@ foreach(characterName in ::Const.Strings.CharacterNames)
 
 // Remove The Robber Baron as a bandit leader name because we have a RobberBaron entity in Reforged
 ::MSU.Array.removeByValue(::Const.Strings.BanditLeaderNames, "The Robber Baron");
+
+::Const.Strings.SouthernNames.extend([
+	"Adil",
+	"Akil",
+	"Akram",
+	"Amin",
+	"Anwar",
+	"Asim",
+	"Badr",
+	"Bashir",
+	"Fahad",
+	"Faisal",
+	"Farid",
+	"Ghassan",
+	"Habib",
+	"Hadi",
+	"Hakim",
+	"Harith",
+	"Idris",
+	"Ihsan",
+	"Imad",
+	"Imran",
+	"Ishaq",
+	"Jafar",
+	"Jamal",
+	"Jibril",
+	"Karim",
+	"Khalid",
+	"Khalil",
+	"Latif",
+	"Layth",
+	"Mahdi",
+	"Mahir",
+	"Marwan",
+	"Munir",
+	"Nadir",
+	"Najeeb",
+	"Naji",
+	"Nawfal",
+	"Qadir",
+	"Qasim",
+	"Qays",
+	"Rafiq",
+	"Saad",
+	"Safwan",
+	"Salah",
+	"Salim",
+	"Samir",
+	"Suhayl",
+	"Talib",
+	"Thabit",
+	"Wahid",
+	"Waqas",
+	"Yasir",
+	"Zafar",
+	"Zayd",
+	"Zeeshan",
+	"Ziyad",
+	"Zuhayr",
+	"Zahid",
+	"Zubayr"
+]);
+
+foreach (name in ::Const.Strings.SouthernNames)
+{
+	::Const.Strings.SouthernNamesLast.push("Ibn " + name);
+	::Const.Strings.SouthernNamesLast.push("Al-" + name);
+}
```

Comment:
The name `Wasi` appears in `SouthernNames` and also in `SouthernNamesLast`. There might be more name overlaps.
Furthermore as described above, two names appear twice in the list bove.

You should remove duplicates after generating this name list, like we do above with the knight names